files/setup.sh

```diff
@@ -27,6 +27,10 @@ VCENTER_DISABLE_TLS=$(vmtoolsd --cmd "info-get guestinfo.ovfEnv" | grep "guestin
 EVENT_PROCESSOR_TYPE=$(vmtoolsd --cmd "info-get guestinfo.ovfEnv" | grep "guestinfo.event_processor_type" | awk -F 'oe:value="' '{print $2}' | awk -F '"' '{print $1}')
 OPENFAAS_PASSWORD=$(vmtoolsd --cmd "info-get guestinfo.ovfEnv" | grep "guestinfo.openfaas_password" | awk -F 'oe:value="' '{print $2}' | awk -F '"' '{print $1}')
 OPENFAAS_ADV_OPTION=$(vmtoolsd --cmd "info-get guestinfo.ovfEnv" | grep "guestinfo.openfaas_advanced_options" | awk -F 'oe:value="' '{print $2}' | awk -F '"' '{print $1}')
+KNATIVE_HOST=$(vmtoolsd --cmd "info-get guestinfo.ovfEnv" | grep "guestinfo.knative_host" | awk -F 'oe:value="' '{print $2}' | awk -F '"' '{print $1}')
+KNATIVE_SCHEME=$(vmtoolsd --cmd "info-get guestinfo.ovfEnv" | grep "guestinfo.knative_scheme" | awk -F 'oe:value="' '{print $2}' | awk -F '"' '{print $1}' | tr [:upper:] [:lower:])
+KNATIVE_DISABLE_TLS=$(vmtoolsd --cmd "info-get guestinfo.ovfEnv" | grep "guestinfo.knative_disable_tls_verification" | awk -F 'oe:value="' '{print $2}' | awk -F '"' '{print $1}' | tr '[:upper:]' '[:lower:]')
```

Comment:
Is the upper/lower required? This comes in as a string or boolean? From the OVF properties it seems it's `boolean`

```
ovf:type="boolean"
```

Reply:
Yes, it comes in as boolean but the values are True/False and I suspect you need the values to be in lower case.

Reply:
That is correct, fine.